jerry-main/main-unix.c

```diff
@@ -107,6 +107,21 @@ assert_handler (const jerry_value_t func_obj_val __attribute__((unused)), /**< f
   }
 } /* assert_handler */
 
+/**
+ * Provide the '__gc' implementation for the engine.
```

Comment:
Double underscore means system function. Print and assert are user functions. I saw other engines use engine_name.gc(), but they provide this as core part of the engine not in a utility application. I can change it to `gc` but that is too short for me.